File: DONGUK/프로그래머스_WEEK 1/멀쩡한 사각형.py
import math

def solution(w,h):
    # 최대 공약수 계산
    gcd = math.gcd(w,h)
    sub = 0
    if gcd == 1:
        sub = w+h-1
    
    else:
        sub = w+h - gcd
    
    answer = w*h - sub
    return answer

File: KIJUNG/PRO_WEEK_02/멀쩡한사각형.cpp
#include <math.h>
#include <iostream>

using namespace std;

int gcd(int a, int b){
    while(b!=0){
        int r = a%b;
        a = b;
        b = r;
    }
    return a;
}

long long solution(int w,int h) {
    long long answer =(long long)w*h - (w+h-gcd(w,h));
    
    return answer;
}